nixos/tests/vscode-remote-ssh.nix

```
@@ -0,0 +1,145 @@
+import ./make-test-python.nix ({ lib, ... }@args: let
+  pkgs = args.pkgs.extend (self: super: {
+    stdenv = super.stdenv.override {
+      config = super.config // {
+        allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [
+          "vscode" "vscode-with-extensions" "vscode-extension-ms-vscode-remote-remote-ssh"
+        ];
+      };
+    };
+  });
+
+  inherit (import ./ssh-keys.nix pkgs) snakeOilPrivateKey snakeOilPublicKey;
+
+  # The commit hash for stable VS Code in Nixpkgs
+  rev = "ee2b180d582a7f601fa6ecfdad8d9fd269ab1884";
```

```suggestion
  rev = "ee2b180d582a7f601fa6ecfdad8d9fd269ab1884";
```
Why stable? Why not current?

---

I've reworded this comment, I was referring to the stable build of VS Code rather than Insiders, as opposed to the stable channel of Nixpkgs